website/src/logged_in/logs/Logs.js

```diff
@@ -76,6 +76,147 @@ const columns = [
             }
         }
     },
+    {
+        field: 'minF&B',
+        headerName: 'Min F&B Per Contract',
+        description: 'F&B Minimum Per Contract',
+        sortable: true,
+        width: 200,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['F&B Minimum Per Contract']}`
+            } catch (error) {
+                return '';
+            }
+        }
+    },
+    {
+        field: 'area',
+        headerName: 'SQFT per Event',
+        description: 'SQFT per Event',
+        sortable: true,
+        width: 175,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['SQFT per Event']}`
+            } catch (error) {
+                return '';
+            }
+        }
+    },
+    {
+        field: 'type',
+        headerName: 'Type',
+        description: 'Event Type',
+        sortable: true,
+        width: 175,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['event_type']}`
+            } catch (error) {
+                return '';
+            }
+        }
+    },
+    {
+        field: 'type-account',
+        headerName: 'Type-account',
+        description: 'Event Type account',
+        sortable: true,
+        width: 175,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['type_account']}`
+            } catch (error) {
+                return '';
+            }
+        }
+    },
+    {
+        field: 'totalDays',
+        headerName: 'Total Days (In-Out)',
+        description: 'Total Days (In-Out)',
+        sortable: true,
+        width: 175,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['total_days']}`
+            } catch (error) {
+                return '';
+            }
+        }
+    },
+    {
+        field: 'isTelecom',
```

Comment:
Column title and description need to be more user friendly.

Reply:
Bump to this comment. We need to make this more user friendly. I'd say `Specialized Equipment` for now. Note: you might need to increase width to ensure the column header is visible

Reply:
Also, you need to convert the `0/1` response to `Yes/No`. We don't want to display binary to a user.